About.html

```diff
@@ -372,49 +380,31 @@ <h3 class="title">Sponsors</h3>
 
   <footer>
     <section class="borderImg footermobile">
-      <div class="row">
-        <div class="col-2 text-center">
-          <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="orange" class="bi bi-boombox-fill mt-3"
-            viewBox="0 0 16 16">
-            <path
-              d="M14.5.5a.5.5 0 0 0-1 0V2H1a1 1 0 0 0-1 1v2h16V3a1 1 0 0 0-1-1h-.5V.5ZM2.5 4a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1Zm2 0a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1Zm7.5-.5a.5.5 0 1 1-1 0 .5.5 0 0 1 1 0Zm1.5.5a.5.5 0 1 1 0-1 .5.5 0 0 1 0 1Zm-7-1h3a.5.5 0 0 1 0 1h-3a.5.5 0 0 1 0-1Zm-2 9a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3Zm.5-1.5a.5.5 0 1 1-1 0 .5.5 0 0 1 1 0Zm6.5 1.5a1.5 1.5 0 1 0 0-3 1.5 1.5 0 0 0 0 3Zm0-1a.5.5 0 1 0 0-1 .5.5 0 0 0 0 1Z" />
-            <path
-              d="M16 6H0v8a1 1 0 0 0 1 1h14a1 1 0 0 0 1-1V6ZM4.5 13a2.5 2.5 0 1 1 0-5 2.5 2.5 0 0 1 0 5Zm7 0a2.5 2.5 0 1 1 0-5 2.5 2.5 0 0 1 0 5Z" />
-          </svg>
-        </div>
-        <div class="col-8 text-center">
-          <h3 class="title">TDM FESTIVAL</h3>
-        </div>
-        <div class="col-2 text-center">
-          <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="orange" class="bi bi-vinyl-fill mt-3"
-            viewBox="0 0 16 16">
-            <path d="M8 6a2 2 0 1 0 0 4 2 2 0 0 0 0-4zm0 3a1 1 0 1 1 0-2 1 1 0 0 1 0 2z" />
-            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM4 8a4 4 0 1 0 8 0 4 4 0 0 0-8 0z" />
-          </svg>
+      <div class="container text-center">
+        <div class="row">
+          <div class="col-6 text-right">
+            <h3 class="title text-justify">TDM <br> FESTIVAL</h3>
+          </div>
+          <div class="col-6 text-left text-white">
+            <p class="mt-3 description">
+              @2022 TDM festival, a festival for evreyone
+            </p>
+          </div>
         </div>
       </div>
```

Comment:
- [ ] You can also change the background color of this section to dark color I think 😀🔨. With all these steps above, your website should change to a lot better look 🍀😉

Your design
![image](https://user-images.githubusercontent.com/69011963/152608930-59d75bfe-5ce8-4685-955c-dc8da9e68ef1.png)

Design from behance
![image](https://user-images.githubusercontent.com/69011963/152609035-08e2479f-28a0-47da-b211-8938d57d4f86.png)